km/Makefile

```diff
@@ -34,8 +35,22 @@ LOCAL_COPTS := -Werror -D_GNU_SOURCE
 LOCAL_LDOPTS := -Wl,--script=km_guest_ldcmd -static-pie
 endif
 
+ifeq (coverage, $(filter coverage,$(MAKECMDGOALS)))
```

Comment:
I think this should be elsif in the if above. Cases being normal, valgrind, coverage.

Reply:
it will be when I move valgrind to use targets too, otherwise we mixing variables and GOALS, hard to read and understand